tests/functional/views/basic_lti_launch_test.py

```diff
@@ -16,30 +16,22 @@
 
 
 class TestBasicLTILaunch:
-    def test_requests_with_no_oauth_signature_are_forbidden(self, app, lti_params):
+    def test_requests_with_no_oauth_signature_are_forbidden(
+        self, lti_params, post_launch
+    ):
         del lti_params["oauth_signature"]
 
-        response = app.post(
-            "/lti_launches",
-            params=lti_params,
-            headers={
-                "Accept": "text/html",
-                "Content-Type": "application/x-www-form-urlencoded",
-            },
+        response = post_launch(
```

Comment:
Switch to a fixture to call the app as needs now to include the get_params.

Reply:
:+1: Saves a bunch of duplication too, and I like to see `response = post_launch(post_params=lti_params, status=403)` because that contains only the parameters that this test actually cares about, and hides away URL and headers that are the same in every test